example_site/src/components/AccountAuthentication.js

```diff
@@ -144,21 +147,29 @@ function CreateAccountForm() {
   };
 
   return (
-    <div className="panel">
-      <AccountInfoForm
-        formTitle="waymark.createAccount()"
-        onSubmit={onSubmit}
-        submitButtonText="Create Account"
-      />
-    </div>
+    <AccountInfoForm
+      onSubmit={onSubmit}
+      formTitle="Create Account"
+      subtitle="All fields are optional"
+      submitButtonText="CREATE ACCOUNT"
+    />
   );
 }
 
 export default function AccountAuthentication() {
   return (
-    <>
-      <CreateAccountForm />
-      <LoginAccountForm />
-    </>
+    <div>
+      <Header 
+        title="Create an Account or Log In"
+        subtitle="To keep track of videos that users have created,
+        you can create a Waymark account with any information you
+        use to identify users on your end."
+      /> 
+      <p></p>
+      <div className='two-columns left'>
```

Comment:
`two-columns` and `left` are both only being used here so you can combine those classes in the stylesheet!